### DISCUSSION TITLE
VLES turbulence model in DAFoam

### DISCUSSION THREAD

**Discussion by erkhans** [mdolab/dafoam#432] [General]
Hello,

Is it possible to implement VLES (Very-large eddy simulation) turbulence model in DAFoam? If yes, could you give some hints on how it can be done?

Thanks!

**Comment by friedenhe** [comment]
Do you want to use VLES for gradient-based optimization? If VLES is an unsteady model, DAFoam will not work because it supports only steady-state adjoint gradient computation at this moment.

**Reply by erkhans** [reply]
I want to use it for FSI.

**Comment by friedenhe** [comment]
Unsteady FSI? Do you have the VLES model implemented in OpenFOAM already and you want to move the implementation to DAFoam? If VLES has not been implemented into OpenFOAM, I suggest you post this question to CFD-online.com.
